bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/types.py

```diff
@@ -47,7 +47,7 @@
 
 names = []
 
-for module in _shared_modules:
+for module in _shared_modules:  # pragma: NO COVER
```

Comment:
Note that this change actually causes the unit-level coverage threshold to **fail** (see #8174).